.ci/build_nccl_rdma_sharp_plugins.sh

```diff
@@ -1,42 +1,37 @@
 #!/bin/bash -leE
 
 SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
-cd ${SCRIPT_DIR}
-. ${SCRIPT_DIR}/settings.sh
+cd "${SCRIPT_DIR}"
+# shellcheck source=settings.sh
+. "${SCRIPT_DIR}/settings.sh"
 
 NCCL_RDMA_SHARP_PLUGINS_DIR="${NCCL_RDMA_SHARP_PLUGINS_DIR:-${WORKSPACE}/_install}"
 echo "INFO: NCCL_RDMA_SHARP_PLUGINS_DIR = ${NCCL_RDMA_SHARP_PLUGINS_DIR}"
 
-cd ${WORKSPACE}
+cd "${WORKSPACE}"
 
-${WORKSPACE}/autogen.sh
-if [ $? -ne 0 ]
-then
+if ! "${WORKSPACE}/autogen.sh"; then
     echo "ERROR: ${WORKSPACE}/autogen.sh failed"
     echo "FAIL"
     exit 1
 fi
 
-${WORKSPACE}/configure \
-    --prefix=${NCCL_RDMA_SHARP_PLUGINS_DIR} \
-    --with-cuda=${CUDA_HOME} \
-    --with-sharp=${HPCX_SHARP_DIR}
-if [ $? -ne 0 ]
-then
+if ! "${WORKSPACE}/configure" \
+    --prefix="${NCCL_RDMA_SHARP_PLUGINS_DIR}" \
+    --with-cuda="${CUDA_HOME}" \
+    --with-sharp="${HPCX_SHARP_DIR}"; then
     echo "ERROR: ${WORKSPACE}/configure failed"
     echo "FAIL"
     exit 1
 fi
 
-make -j install
-if [ $? -ne 0 ]
-then
+if ! make -j install; then
     echo "ERROR: 'make -j install' failed"
     echo "FAIL"
     exit 1
 fi
 
 echo "INFO: ${NCCL_RDMA_SHARP_PLUGINS_DIR}:"
-find ${NCCL_RDMA_SHARP_PLUGINS_DIR} -type f
+find "${NCCL_RDMA_SHARP_PLUGINS_DIR}" -type f
```

Comment:
What is the purpose of this line?

Reply:
Just for debug purposes to see the content of ${NCCL_RDMA_SHARP_PLUGINS_DIR} folder (built binaries).

Reply:
Could you please add this comment to the script?

Reply:
Fixed.